api/app/signals/settings/base.py

```diff
@@ -230,8 +230,10 @@
 
 EMAIL_APPTIMIZE_INTEGRATION_ADDRESS = os.getenv('EMAIL_APPTIMIZE_INTEGRATION_ADDRESS', None)
 EMAIL_FLEX_HORECA_INTEGRATION_ADDRESS = os.getenv('EMAIL_FLEX_HORECA_INTEGRATION_ADDRESS', None)
-EMAIL_FLEX_HORECA_WEEKDAYS = os.getenv('EMAIL_FLEX_HORECA_WEEKDAYS',
-                                       '5,6')  # Default: friday, saterday
+EMAIL_FLEX_HORECA_WEEKDAYS = os.getenv(
+    'EMAIL_FLEX_HORECA_WEEKDAYS', '5,6,7')  # friday, saterday, sunday
+EMAIL_FLEX_HORECA_END_TIME = os.getenv(
+    'EMAIL_FLEX_HORECA_END_TIME', '4,0')  # 04:00 o'clock
```

Comment:
Why not use `04:00` as time setting directly? Now we have a comment explaning the format and extra complications parsing and merging two settings into a time object.

Reply:
Agreed, fixed